src/EventListener/ExceptionListener.php

```diff
@@ -24,6 +25,23 @@
  */
 final class ExceptionListener extends BaseExceptionListener
 {
+    /**
+     * @var array
+     */
+    private $exceptionToStatus;
+
+    /**
+     * @param mixed           $controller
+     * @param LoggerInterface $logger
+     * @param array           $exceptionToStatus
+     */
+    public function __construct($controller, LoggerInterface $logger = null, array $exceptionToStatus = null)
+    {
+        parent::__construct($controller, $logger);
+
+        $this->exceptionToStatus = $exceptionToStatus ?? [];
```

Comment:
Can't you set the default value to `[]` directly?

Reply:
hm, i always use `null` for optional arguments. here is why:
`function test(array $required, array $opt1 = [], array $opt2 = [])` if i want to call `test` without the first optional parameter i have to do it like this: `test([], [], [])`. that means, i have to know that the 2nd parameter has to be an array and that it is an empty array. i cant just "skip" it, by passing `null` like so `test([], null, [])`.
i agree that with arrays that is not such a big deal, but consider the following:
`test2(int $a = SOME_LONG_CONSTANT, int $b = 0)`, if i want to set `b`, i have to use `SOME_LONG_CONSTANT` insteadof just "skipping" the param with `null`.

Reply:
The IDE helps for that (but yes, we need named arguments in PHP!).

We use arrays, boolean and so on everywhere else in the framework, so I would prefer you change it for consistency.